### DISCUSSION TITLE
M5 AtomS3

### DISCUSSION THREAD

**Discussion by macgyverek** [headtracker/HeadTracker#184] [Q&A]
I found on your project that you have defined esp32c3 and m5stick_plus. Headtracker working on this devices? Did you tested? I have M5 AtomS3 and I can (try) add device and overlays for your project on Zephyr. Now I have Zephyr setup for esp32c3 ;)

**Comment by dlktdr** [comment]
M5cstick was close. No display but imu, gui connection, flash was working. Bluetooth was sort of, but crashed from what i remember.

Haven't wrote ppm in/out or aux serial yet. Aux serial should be simple.

PPM really wanted to do it with the RMT feature on ESP but since there is no driver in zephyr yet isn't simple.

Was thinking of just doing a generic interrupt, but might be some stutter
